src/honeybadger/core.clj

```diff
@@ -28,6 +28,13 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(defn ex-chain
+  "Return all exceptions in the exception chain."
+  [e]
+  (->> e
+       (iterate #(try (.getCause %) (catch NullPointerException _)))
```

Comment:
Is there a need for this that can't be obviated by simply checking if the value is `nil`? (I'd rather not cause and catch exceptions in non-exceptional circumstances.)